### PR TITLE
Modified PR Template

### LINKED ISSUE #2021: Update PR Template
**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by sachin235**
Fixes #2021

Changes: Added description for Screenshots and made minor changes.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this Pull Request and if possible provide links chowcasing the changes.]
```
Comment (AakashMallik) [inline .github/PULL_REQUEST_TEMPLATE]: chowcasing?? Hmm.. please double check before pushing

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
 Demo Link: https://pr-[ADD_PULL_REQUEST_NUMBER_HERE]-fossasia-susi-web-chat.surge.sh
 
-Screenshots for the change: 
+Screenshots for the change: [Add here screenshots depicting the changes.]
```
Comment (AakashMallik) [inline .github/PULL_REQUEST_TEMPLATE]: Add screenshots depicting the changes

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: Sorry Sir.
Thanks for bringing it to notice!!

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: I've made the suggested changes.

File: .github/PULL_REQUEST_TEMPLATE [hunk 3]
```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this Pull Request and if possible provide links showcasing the changes.]
 
 Demo Link: https://pr-[ADD_PULL_REQUEST_NUMBER_HERE]-fossasia-susi-web-chat.surge.sh
 
-Screenshots for the change: 
+Screenshots for the change: [Add screenshots depicting the changes.]
```
Comment (AakashMallik) [inline .github/PULL_REQUEST_TEMPLATE]: **for the** -> of the

File: .github/PULL_REQUEST_TEMPLATE [hunk 4]
```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this Pull Request and if possible provide links showcasing the changes.]
```
Comment (AakashMallik) [inline .github/PULL_REQUEST_TEMPLATE]: Any particular reason for upper case **pull request**?

File: .github/PULL_REQUEST_TEMPLATE (hunk 3 above)
Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: OK sir.

File: .github/PULL_REQUEST_TEMPLATE (hunk 4 above)
Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: It was to emphasize on PR, but I'll change it to lower case. That seems better.

**Comment by sachin235** [comment]
Done, updated the branch

File: .github/PULL_REQUEST_TEMPLATE [hunk 5]
```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this pull request and if possible provide links showcasing the changes.]
```
Comment (fragm3) [inline .github/PULL_REQUEST_TEMPLATE]: Make it were

Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: Made the required changes as suggested.

**Pull request merged**